frontend/src/expenditurePage/ExpenditurePage.js

```diff
@@ -0,0 +1,32 @@
+import React, {useContext} from 'react';
+import EventContext from "../contexts/EventContext";
+import {useHistory, useParams} from "react-router-dom";
+import Card from "@material-ui/core/Card";
+import {Button} from "@material-ui/core";
+
+export default function ExpenditurePage() {
+  const { events } = useContext(EventContext);
+  const { id, expenditureId } = useParams();
+  const event = events.find((event) => event.id === id);
+  const expenditure = event?.expenditures.find((exp) => exp.id === expenditureId)
```

Comment:
Nice chaining, in combination with conditional rendering, makes for robust code 👍